pom.xml

```diff
@@ -18,8 +18,10 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <spring.springframework.version>5.0.1.RELEASE</spring.springframework.version>
-        <spring.boot.starter.test.version>2.0.0.RELEASE</spring.boot.starter.test.version>
         <spring.data.version>2.0.0.RELEASE</spring.data.version>
+        <spring.boot.version>2.0.0.RELEASE</spring.boot.version>
```

Comment:
are those spring boot version all under spring framework dependencies? can we import that dependencies management?

Reply:
thanks, dependencies management looks more clear here.